composer.json

```diff
@@ -203,10 +206,12 @@
         "git-hook-install": "@php .github/contrib/install.php",
         "git-hook-uninstall": "@php .github/contrib/uninstall.php"
     },
+    "scripts-descriptions": {
+      "test-all": "Launch all PHPUnit test suites",
```

Comment:
Why two spaces here when the whole file is four :D

Reply:
it's my PHPStorm setup :p, I'll revert that: let's see how @eternoendless will react to this pull request first ;)

Reply:
As I said in a mail few days ago, we need to communicate before merging this kind of work 😺

Reply:
solved!